test/cmdlineTests/model_checker_solvers_smtlib2/err

```diff
@@ -0,0 +1,3 @@
+Warning: CHC analysis was not possible since no Horn solver was found and enabled.
+
+Warning: BMC analysis was not possible since no SMT solver (Z3 or CVC4) was found and enabled.
```

Comment:
Yea, what I meant before was `no Horn solver was (found and enabled).`

Reply:
@cameel I still prefer the `and `version here: a solver must be both found AND enabled to be used, that's why the previous phrasing.

Reply:
Well, reading it as `not (found and enabled)` makes total sense, but for some reason putting it into English makes it look less unambiguous to me. That might be just subjective though. The phrasing is correct so I'm fine with keeping it since it's just a small detail.

Alternatively, how about rephrasing it so that we don't need AND or OR:
```
Warning: BMC analysis was not possible. No SMT solver (Z3 or CVC4) available. None of the installed solvers was enabled.
```
```
Warning: BMC analysis was not possible. No SMT solver (Z3 or CVC4) available. None of the enabled solvers was detected.
```

Reply:
That makes sense